bedrock/base/templates/includes/stat_counter.html

```diff
@@ -0,0 +1,7 @@
+<noscript>
+  <div class="statcounter">
+    <a title="site stats" href="http://statcounter.com/" target="_blank">
+      <img class="statcounter"src="//c.statcounter.com/11297457/0/d8541690/1/" alt="sitestats">
```

Comment:
We should keep our specific ID (which is `d8541690`, afaict) in the `settings` object to avoid having it hard-coded in these templates.

I suggest copying the way we do [`OPTIMIZELY_PROJECT_ID` in `base.py`](https://github.com/mozilla/bedrock/blob/master/bedrock/settings/base.py#L1113). (Feel free to place `STATCOUNTER_ID` just after `OPTIMIZELY_PROJECT_ID`.)